src/components/protocol_handler/test/include/protocol_handler/control_message_matcher.h

```diff
@@ -153,7 +154,8 @@ MATCHER_P4(ControlMessage,
     *result_listener << "Message with " << data_vector.size()
                      << " byte data : 0x";
     for (size_t i = 0u; i < data_vector.size(); ++i) {
-      *result_listener << std::hex << static_cast<int>(data_vector[i]);
+      *result_listener << std::setw(2) << std::setfill('0') << std::hex
+                       << static_cast<int>(data_vector[i]);
```

Comment:
Unrelated changes

Reply:
This is to fix (or enhance) message dumping. I can remove it and probably create a separate PR.

Reply:
The change is removed by commit https://github.com/smartdevicelink/sdl_core/pull/2164/commits/b09936e5c232765c873545bc34a48f58ac6e60e6. A new PR is filed for this: https://github.com/smartdevicelink/sdl_core/pull/2311.